import_export/admin.py

```diff
@@ -560,10 +560,20 @@ def export_admin_action(self, request, queryset):
                 self.get_export_filename(request, queryset, file_format),
             )
             return response
-    export_admin_action.short_description = _(
-        'Export selected %(verbose_name_plural)s')
 
-    actions = admin.ModelAdmin.actions + [export_admin_action]
+    def get_actions(self, request):
+        """Returns the dictionary of Admin actions after adding the
```

Comment:
I'd say we follow current code style until `black` is applied everywhere, please add a new line after `"""`